code/datums/diseases/tuberculosis.dm

```diff
@@ -0,0 +1,64 @@
+/datum/disease/Tuberculosis
```

Comment:
capital at the start of the path? that's... uncommon.
best to change it for consistency, unless of course the other disease do that in which case wat.